base/linalg/cholesky.jl

```diff
@@ -161,6 +161,25 @@ end
 
 Compute the Cholesky factorization of a positive definite matrix `A`
 and return the UpperTriangular matrix `U` such that `A = U'U`.
+
+**Example**
```

Comment:
Yep, you're right. I should have checked the docs instead of just looking at the existing example headings in the file. My bad, fixed it.